pkg/roachprod/install/cockroach.go

```diff
@@ -903,21 +907,25 @@ func (c *SyncedCluster) generateStartArgs(
 
 	// if neither --log nor --log-config-file are present
 	if idx1 == -1 && idx2 == -1 {
-		loggingConfig, err := execLoggingTemplate(loggingTemplateData{
-			LogDir: logDir,
-		})
-		if err != nil {
-			return nil, errors.Wrap(err, "failed rendering logging template")
-		}
+		if !startOpts.EnableFluentSink {
+			args = append(args, "--log", `file-defaults: {dir: '`+logDir+`', exit-on-error: false}`)
+		} else {
+			loggingConfig, err := execLoggingTemplate(loggingTemplateData{
+				LogDir: logDir,
+			})
+			if err != nil {
+				return nil, errors.Wrap(err, "failed rendering logging template")
+			}
 
-		loggingConfigFile := fmt.Sprintf("cockroachdb-logging%s.yaml",
-			virtualClusterDirSuffix(startOpts.VirtualClusterName, startOpts.SQLInstance))
+			loggingConfigFile := fmt.Sprintf("cockroachdb-logging%s.yaml",
+				virtualClusterDirSuffix(startOpts.VirtualClusterName, startOpts.SQLInstance))
 
-		if err := c.PutString(ctx, l, c.Nodes, loggingConfig, loggingConfigFile, 0644); err != nil {
-			return nil, errors.Wrap(err, "failed writing remote logging configuration: %w")
-		}
+			if err := c.PutString(ctx, l, Nodes{node}, loggingConfig, loggingConfigFile, 0644); err != nil {
```

Comment:
This was changed since our containing function `generateStartArgs` already takes a single node, not the cluster entirely. We were running this copy operation across all nodes in the cluster N times. Now we just run the copy operation once per node.